utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -434,7 +434,10 @@ Status PessimisticTransactionDB::Merge(const WriteOptions& options,
 }
 
 Status PessimisticTransactionDB::Write(const WriteOptions& opts,
-                                       WriteBatch* updates) {
+                                       WriteBatch* updates, bool skip_cc) {
+  if (skip_cc) {
```

Comment:
Similarly, a comment about skip_cc skipping locking, so that we call the underlying db directly.